### PR TITLE
102 localize images

### PR DESCRIPTION
**Pull request by philcryer**
I've localized all the 'mainimage' images for everyone, updated names where needed, followed naming standards, also updated the images pulled from 'url' section, in the case of multiple user I did: /images/User_name.jpg for mainimage, then /images/User_name2.jpg for any additional, and rewrote the 'url' in that block of the json. So all people json and images localized. Would like to contribute more, really happy to see this project.

### CODE REVIEW

**Comment by mauvehed** [comment]
Thank you, @philcryer! I'll test these locally to verify our automated checks are working as intended before landing this.

**Comment by philcryer** [comment]
Sounds good @mauvehed - how do you check to make sure the automated checks are working? In other words would I need to view this code via a webserver and manually click all the links I touched just to be sure I got all the image and names correct? Might want to add to CONTRIBUTING.md if so, as I'd like to help more, and make PR smoother. Thanks!

File: people/barnabyjack.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
 	"birth" : "1977",
 	"death" : "2013",
 	"issue" : "4",
-	"mainimage" : "/images/y3t1_Art-Barnaby_Jack.png",
```
Comment (mubix) [inline people/barnabyjack.json]: First of, this is all great, thank you for doing this. One small nit pick is that we want y3ti's art to keep it's designation for attributions sake.

Comment (philcryer) [reply people/barnabyjack.json]: Cool, didn't realize it, fixed that with: https://github.com/restincode/restincode/pull/129/commits/e5dada05bf14b4f1c6e241174e6b4440b9d18b62

**Pull request merged**